src/FocusAPI/FocusAPI.csproj

```diff
@@ -10,12 +10,23 @@
     <DockerfileContext>..\..</DockerfileContext>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'https' " />
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'http' " />
```

Comment:
Could you explain the addition of these two property groups?

Reply:
Auto generated pkg that slipped into commit. Good catch!